.travis.yml

```diff
@@ -1,11 +1,15 @@
-sudo: false
+sudo: required
```

Comment:
@gurneyalex that's why then: https://travis-ci.org/OCA/knowledge/jobs/148364064 - the docker image still has an older version of tesseract that doesn't work well with the module